news/1256.bugfix

```diff
@@ -0,0 +1 @@
+Change the checkout of a not existing branch to master [ale-rt]
```

Comment:
@ale-rt I'd remove the changelog here. That's an "internal" change that currently Towncrier does not have a category for.

Reply:
@ale-rt I will merge this PR now and remove the changelog.